CHANGELOG.md

```diff
@@ -66,6 +66,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#623](https://github.com/nix-rust/nix/pull/623))
 - Multiple constants related to the termios API have now been properly defined for
   all supported platforms. ([#527](https://github.com/nix-rust/nix/pull/527))
+- Fixed field types of `sys::socket::ffi::{msghdr, cmsghdr}` on
+  non-Linux platforms.  ([#](https://...))
```

Comment:
You can fill this in at this point. It won't change between now and when it's committed.